### PR TITLE
DG -  add, select features

### LINKED ISSUE #119: Write DG for add notebook
make UML diagram for that as well

### LINKED ISSUE #120: Write DG for add task
make UML diagram as well...

### LINKED ISSUE #121: Write DG for select implementation

### PR DESCRIPTION
**Pull request by franceneee**

### CODE REVIEW

**Comment by franceneee** [comment]
Should fix #119, #120, #121.

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -209,16 +209,36 @@ decisions.
...
- `Task`s, and a few helper methods. 
+`TaskList` is implemented to manage and store the tasks input by the user. It comprises an `ArrayList` list of
+ `Task`s, and a few helper methods.
 
 This means that multiple operations such as addition and deletion can be done on a `Task`, without affecting
 the contents of other `Task` in the `TaskList`.
-{Introduce how the addition command works}
+
+<br>
+
```
Comment (Lusi711) [inline docs/DeveloperGuide.md]: Extra new line here

Comment (Lusi711) [reply docs/DeveloperGuide.md]: <br> probably redundant

Comment (franceneee) [reply docs/DeveloperGuide.md]: honestly yea, the `<br>` is doing the magic xD

**Review by yAOwzers — APPROVED** [review]

**Pull request merged**